saleor/core/permissions.py

```diff
@@ -52,6 +52,10 @@ class ProductPermissions(BasePermissionEnum):
     MANAGE_PRODUCTS = "product.manage_products"
 
 
+class ProductTypePermissions(BasePermissionEnum):
```

Comment:
Just to confirm. Is the name of class correct? It says `ProductType` but I see it is related to attributes also 🤔

Reply:
I know, but I have to join it with one model in the product models, and I decided to join it with `ProductType` as attributes are connected to it. Because of that, I named the class `ProductTypePermissions`.